.github/workflows/release.yaml

```diff
@@ -3,26 +3,32 @@ name: Release
 "on":
   workflow_dispatch: {}
```

Comment:
We could think about to run this workflow on tags or might this collidate w/ `mvn release`?

Reply:
In other maven projects in @kokuwaio we already use release plugin with **release:prepare** and **release:perform**. Building on tags  skips the **release:prepare** part and you have to set the version from tag to **pom.xml** before performing the release. Because this is more complicated i prefer **release:prepare** with **release:perform**.